domains/merkleroots.go

```diff
@@ -28,3 +28,13 @@ type MerkleRootConfirmation struct {
 	Hash         string                      `json:"hash,omitempty"`
 	Confirmation MerkleRootConfirmationState `json:"confirmation"`
 }
+
+// MerkleRootsResponse is the response object that should be returned from the database when
+// requested from merkleroots api endpoint
+type MerkleRootsResponse struct {
```

Comment:
Ehhh, We have a mess with contextes of the structs. IMHO it shouldn't belong to domain, but rather to endpoint. 
But this is part of bigger discussion.